### PR TITLE
Add command to connect user account to MS Teams

### LINKED ISSUE #66: There Should be a separate command for Connect.
Currently when user type /mstmeetings Start then if user is not connected then user will get the link to connect so there should be separate command for connect same as other plugins.

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
- Added command to connect user account to MS Teams
- Updated help command

#### Ticket Link
Fixes #66 

#### What to test?
- Run the slash command `/mstmeetings connect`
- Check if user is able to connect by clicking on the provided link
- Run the slash command `/mstmeetings connect` again after connecting, the user should get the ephemeral post `User already connected to MS Teams Meetings`

#### Checklist
- [x] Completed dev testing
- [x] `make test` Ran test cases and ensured they are passing
- [x] `make check-style` Ran style check and ensured both webapp and server pass the checks

### CODE REVIEW

File: server/plugin.go [hunk 1]
```diff
@@ -25,8 +25,6 @@ const (
 type Plugin struct {
 	plugin.MattermostPlugin
 
-	client *pluginapi.Client
-
```
Comment (mickmister) [inline server/plugin.go:29]: I suppose this isn't used anywhere?

File: server/command.go [hunk 2]
```diff
@@ -13,7 +13,8 @@ import (
 
 const (
 	commandHelp = `* |/mstmeetings start| - Start an MS Teams meeting.
-	* |/mstmeetings disconnect| - Disconnect from Mattermost`
+* |/mstmeetings disconnect| - Disconnect from Mattermost
+* |/mstmeetings connect| - Connect to MS Teams meeting`
```
Comment (mickmister) [inline server/command.go:17]:
```suggestion
* |/mstmeetings connect| - Connect to MS Teams meeting`
* |/mstmeetings disconnect| - Disconnect from Mattermost
```

**Review by mickmister — APPROVED** [review]

File: server/plugin.go (hunk 1 above)
Comment (ayusht2810) [reply server/plugin.go:29]: Yes

**Review by larkox — APPROVED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]
This PR has been tested for the following scenarios:

- Check the newly added command to connect user account to MS Teams
- Check the updated help command

The PR was working fine for both the conditions. LGTM. Approved

**Pull request merged**